### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_ScreenReader : Narrator is announcing extra information when the focus lands on "Skip to main content" Link.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#179]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#WCAG1.3.1;#DesktopWeb;#Win11;#ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)

### Repro Steps:
1. Hit the URL: https://microsoft.github.io/30daysof/
2. Turn on the Narrator Screen reader. Tab till "Skip to main content" link.
3. Observe that Narrator is announcing Extra information when the focus lands on the Skip to main content link.
### Actual:

While Navigating the screen with Narrator screen reader, Narrator is announcing "theme.common.skiptomaincontent" link when focus lands on Skip to main content link.
Note – This issue is NOT reproducing with NVDA screen reader.

### Expected:
Narrator Screen reader should announce only "Skip to main content" link.

### User Impact:
Screen reader user will be confused while accessing skip to main content link.

### Attachment:
![skip main](https://user-images.githubusercontent.com/85863895/209611945-956d4d97-44eb-45a9-9ad2-c4617b64e77a.gif)

**Comment by nitya** [comment]
@Ashutoshsahoo164  we have decided to unpublish the site and deactivate it from internal listings. Can you verify and close this issue now that website is no longer active?

**Comment by Ashutoshsahoo164** [comment]
As per the above comments , closing this bug.
